src/pages/Quiz/Quiz.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+
+import { useQuiz } from "../../context";
+import { QuestionBank } from "../../database/Quiz.type";
+import { quizBeginner } from "../../database/quizBeginner";
+import { GamePlay } from "./GamePlay";
+import { Instructions } from "./Instructions";
+import { GameResults } from "./GameResults";
+import { QuizDispatchTypeEnum } from "../../utils";
+
+export function Quiz() {
+   const {
+      quizState: { selectedQuiz, currentQuestion },
+      quizDispatch,
+   } = useQuiz();
+
+   const [game, setGame] = useState<QuestionBank | null>(null);
+
+   const [instructions, setInstructions] = useState<boolean>(true);
+
+   const [start, setStart] = useState<boolean>(false);
+
+   const [showResult, setShowResult] = useState<boolean>(false);
+
```

Comment:
If more than 4 states, you can refactore the code into useReducer

Reply:
Thanks for this suggestion and have implemented the same